doc/dynamo0p3.rst

```diff
@@ -1246,35 +1246,35 @@ where:
 Boundary Conditions
 -------------------
 
-In the dynamo0.3 API, boundary conditions for a field can be enforced
-by the algorithm developer by calling a particular Kernel called
-``enforce_bc_type``. This kernel takes a field as input and applies
+In the dynamo0.3 API, boundary conditions for a field or LMA operator can
```

Comment:
Agreed, the problem is that it potentially involves a lot of changes as the dynamo0.3 name is used all over the place. I think we should rename it the LFRic1.0 API. We could possibly do this before the 1.4.0 release, but it would cause a delay.